db/migrate/20230328223126_create_user.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
as required you should have a migration file for creating a users table as seen below
![image](https://user-images.githubusercontent.com/60612329/228525717-bfc73e22-64c2-4431-95d0-9d7b55118f1b.png)